packages/host/app/components/operator-mode/container.gts

```diff
@@ -608,6 +610,20 @@ export default class OperatorModeContainer extends Component<Signature> {
     return this.operatorModeStateService.state?.stacks.flat() ?? [];
   }
 
+  get cardForCodeMode() {
+    // Last card in rightmost stack
+    return (
+      this.allStackItems
+        .filter((item) => item.type === 'card')
```

Comment:
Would a [type predicate work](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) to avoid the `ts-ignore`?

Reply:
Yeah you're right we shouldn't be adding ts-ignores so casually. I'm leaning towards just casting it:

```
return (
      this.allStackItems.filter(
        (item) => item.type === 'card',
      ) as CardStackItem[]
    ).reverse()[0].card;
```